rmpv/src/ext/de.rs

```diff
@@ -534,6 +596,98 @@ impl<'de> Deserializer<'de> for &'de ValueRef<'de> {
     }
 }
 
+enum ExtData<'de> {
```

Comment:
This looks like a re-implementation of `std::borrow::Cow<'de, [u8]>`. I think it should be directly replaceable with that?